docs/Spring Boot/5_Step_To_Make_Gradle_Configuration_Extreme_Clean_In_A_Multi-Module_Project.md

```diff
@@ -0,0 +1,216 @@
+# 5 Step To Make Gradle Configuration Extreme Clean In a Multi-Module Project
```

Comment:
⚠️ **[vale]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Readability.SMOG] Try to keep the SMOG grade (11.58) below 10.